restservice/src/main/java/org/itevents/controller/UserRestController.java

```diff
@@ -85,4 +85,11 @@ public void deactivateSubscription() {
         User user = userService.getUser(userId);
         return eventService.getEventsByUser(user);
     }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/activate/{otp}")
```

Comment:
Yes, you are right we need use `POST` but when we click at OTP-link in email our browser sends query using `GET` request method. Therefore you will get `HTTP ERROR 405 Request method 'GET' not supported` when you try to go via `/users/activate/123` from email.